phpBB/includes/functions_compatibility.php

```diff
@@ -162,7 +162,7 @@ function cache_moderators()
 /**
 * Removes moderators and administrators from foe lists.
 *
-* @deprecated 3.1.0 (To be removed: 3.3.0)
```

Comment:
yeah, it was the idea of @Nicofuma iirc. And since it only means that we don't delete something i guess it is a fairly good one.